bcp/client/src/webpack/helper/map-style.js

```diff
@@ -471,3 +462,151 @@ const locationUnclassifiedIcon = L.icon({
     popupAnchor: [0, -12]
 });
 
+const equipmentIconMarker = (iconColor, markerColor) => L.ExtraMarkers.icon({
+    innerHTML: equipmentIconHtml(iconColor, markerColor),
+    svg: true
+});
+const planeIconMarker = (iconColor, markerColor) => L.ExtraMarkers.icon({
+    innerHTML: planeIconHtml(iconColor, markerColor),
+    svg: true
+});
+const planeIconHtml = (iconColor, markerColor) => {
+    return `<svg class="fa-content" width="15" height="15" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512"><path fill='${iconColor}' d="M476 3.2L12.5 270.6c-18.1 10.4-15.8 35.6 2.2 43.2L121 358.4l287.3-253.2c5.5-4.9 13.3 2.6 8.6 8.3L176 407v80.5c0 23.6 28.5 32.9 42.5 15.8L282 426l124.6 52.2c14.2 6 30.4-2.9 33-18.2l72-432C515 7.8 493.3-6.8 476 3.2z"/></svg><svg class="truckContainer" width="35" height="45" viewBox="0 0 35 45" xmlns="http://www.w3.org/2000/svg"><path fill='${markerColor}' d="M28.205 3.217H6.777c-2.367 0-4.286 1.87-4.286 4.179v19.847c0 2.308 1.919 4.179 4.286 4.179h5.357l5.337 13.58 5.377-13.58h5.357c2.366 0 4.285-1.87 4.285-4.179V7.396c0-2.308-1.919-4.179-4.285-4.179"  /><g opacity=".15" transform="matrix(1.0714 0 0 -1.0714 -233.22 146.783)"><path d="M244 134h-20c-2.209 0-4-1.746-4-3.9v-18.525c0-2.154 1.791-3.9 4-3.9h5L233.982 95 239 107.675h5c2.209 0 4 1.746 4 3.9V130.1c0 2.154-1.791 3.9-4 3.9m0-1c1.654 0 3-1.301 3-2.9v-18.525c0-1.599-1.346-2.9-3-2.9h-5.68l-.25-.632-4.084-10.318-4.055 10.316-.249.634H224c-1.654 0-3 1.301-3 2.9V130.1c0 1.599 1.346 2.9 3 2.9h20" fill="#231f20" /></g></svg>`;
```

Comment:
Why did this and the equipment icon need to be inlined?

Reply:
In order to import these as a string, we would need to use a loader (e.g. https://v4.webpack.js.org/loaders/raw-loader/). It should then be possible to style them with css.

Reply:
Actually, it looks like we might be able to avoid using loaders in some cases https://webpack.js.org/guides/asset-modules/

Reply:
Nice, we could simply add this rule and svgs would be imported as strings
`{
       test: /\.svg/,
       type: 'asset/source',
 }`